src/Bicep.Core/Analyzers/Linter/Rules/BCPL1000.cs

```diff
@@ -0,0 +1,42 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using Bicep.Core.Analyzers.Interfaces;
+using Bicep.Core.Parsing;
+using Bicep.Core.Semantics;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Bicep.Core.Analyzers.Linter.Rules
+{
+    public sealed class BCPL1000 : LinterRule
+    {
+        public BCPL1000() : base(
+            code: "BCPL1000",
+            ruleName: "Parameters Required",
+            description: "A valid template must be parameterized.",
```

Comment:
I get this is a TTK rule that we're moving over, but I do have to say, I really don't understand this rule. I feel like it would annoy the crap out of me - especially as I presume it'll show up on every new blank doc.

Is there a notion of enabled/disabled by default, and if so, should this rule be disabled by default?

Reply:
There is a notion of hardcoded defaults in the constructor for each rule as well as defaults provided in the bicepsettings.json

Reply:
This rule has been removed per bmoore